File: Just_codes/Python/area_rectangle.py
class rectangle():
    def __init__(self,length,breadth):
        self.length=length
        self.breadth=breadth
    def area(self):
        return self.length*self.breadth

a=int(input('Enter a length of rectangle: '))
b=int(input('Enter a breadth of rectangle: '))
obj=rectangle(a,b)

print("The area of rectangle is: ",obj.area())

File: Just_codes/C/area_rectangle.c
#include <stdio.h>

int main(){
    
    // Declare a Variable
    
    float length,width,rectangle;
    printf("Enter a length: ");
    scanf("%f",&length);
    printf("Enter a width: ");
    scanf("%f",&width);
    
    // Using the formula for area of rectangle
    
    rectangle = length * width;
    printf("Area of rectangle = %f sq. units",rectangle);
    
    
}
